Exercise_2/interactive_command_list

```diff
@@ -2,9 +2,9 @@
 srun --partition=express --nodes=1 --cpus-per-task=2 --pty --time=00:30:00 /bin/bash
 
 source ~/spack/share/spack/setup-env.sh
```

Comment:
You should also include 
module load python/3.8.1
before sourcing the environment because the new version wants a new version of python.

Reply:
You should include `module load python/3.8.1` because the latest version of spack will not run with the system python.